### DISCUSSION TITLE
Templater + GPT3

### DISCUSSION THREAD

**Discussion by louis030195** [SilentVoid13/Templater#772] [Help]
Hi, I have created https://github.com/louis030195/obsidian-ava which allows you to be assisted by a language model such as [GPT3](https://openai.com/api/) when writing.

GPT3 allows you to program things in 5 lines of natural language rather than 50 lines of programming language.

My plan is to be able to customize how the prompt is built with templates, in the first order I thought about chaining templates using https://handlebarsjs.com/, but Templater seems more interesting.

I am not very familiar with Templater, maybe someone could help and give some suggestions on how I could use it in "obsidian-ava"?

**Comment by welpdx** [comment]
Not sure what you want to do exactly. 
All I know is that Templater uses the [Eta js](https://eta.js.org/) templating engine. You can learn alot about Templater by looking at the [documentation](https://silentvoid13.github.io/Templater/syntax.html).
